department-of-veterans-affairs/caseflow
# ryan/APPEALS-29952

Resolves [Receipt Date Filter Dates Need Label "Receipt Date"](https://jira.devops.va.gov/browse/APPEALS-29952)

# Description
The recieptdate filter previously showed labels for  the to, from, and, on dropdowns. These changes make it so that the labels are only displayed on the between these dates option of the dropdown.
## Acceptance Criteria

## Testing Plan

1. Navigate to caseflow and choose a user that can access reader (BVADWISE works)
2. Navigate to the reader section of caseflow.
3. Click the reciept date filter Icon and notice that when the filter is set to 'before', 'after', or, 'on' this date, there is no labels displayed for the name of the filter.
4. Change the filter to between these dates. The 'to' and 'from' labels are now display here, and only here.

# Frontend
## User Facing Changes

 BEFORE|AFTER
<img width="1315" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/c45376f2-47d0-438b-a680-c743e7262485">|
<img width="1361" alt="image" src="https://github.com/department-of-veterans-affairs/caseflow/assets/110805785/873ffdc7-88b9-4da4-a6b4-fa8d5c1739e2">
